router/registry_handler_test.go

```diff
@@ -0,0 +1,46 @@
+package router
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestRegistry(t *testing.T) {
+	r := NewTestRouter(withLiquid)
+	resp := mintRequest(r, liquidAddress, assetQuantity, name, ticker)
+	resp = registryRequest(r, []interface{}{})
+
+	if resp.Code != http.StatusOK {
+		t.Fatalf("Expected status %d, got: %d\n", http.StatusOK, resp.Code)
+	}
+
```

Comment:
check also that the returned repsonse of the POST /registry will give back the right asset